src/Compilers/CSharp/Portable/Syntax/ParameterSyntax.cs

```diff
@@ -20,7 +16,7 @@ internal bool IsArgList
 
         public ParameterSyntax Update(SyntaxList<AttributeListSyntax> attributeLists, SyntaxTokenList modifiers, TypeSyntax type, SyntaxToken identifier, EqualsValueClauseSyntax @default)
```

Comment:
>@default [](http://example.com/codeflow?start=178&length=8)

Not part of this PR, but this parameter seems incorrectly named. Should this be `initializer` instead?